server-kotlin/src/main/resources/application.properties

```diff
@@ -17,5 +17,13 @@ city-request.appName=CityVizor
 city-sync.instanceUrls.prague=https://cityvizor.praha.eu:8080
 # Fulltext search
 fulltextSearch.indexRefreshEnabled=true
+# Budgets download
+budgets.frequency=PT60M
+# url of cities as defined in app.profiles, column url
+budgets.cityUrls = mestocernosice, ub, nmnm
```

Comment:
This is really fragile. If the order does not match baseSourceUrls, it will break in a really ugly way. Consider using something like this https://www.baeldung.com/configuration-properties-in-spring-boot#nested-properties